gdpr/anonymizers/model_anonymizers.py

```diff
@@ -45,19 +46,28 @@ def __new__(cls, name, bases, attrs):
         for name, obj in attrs.items():
             if isinstance(obj, FieldAnonymizer):
                 fields[name] = obj
+
         new_obj.fields = fields
 
+        anonymizers = getattr(new_obj, 'anonymizers', {})
+        for name, obj in attrs.items():
+            if isinstance(obj, BaseAnonymizer):
+                anonymizers[name] = obj
+        new_obj.anonymizers = anonymizers
+
         if not getattr(new_obj.Meta, 'abstract', False):
             anonymizer_register.register(new_obj.Meta.model, new_obj)
 
         return new_obj
 
 
-class ModelAnonymizerBase(metaclass=ModelAnonymizerMeta):
+class ModelAnonymizerBase(BaseAnonymizer, metaclass=ModelAnonymizerMeta):
     can_anonymize_qs: bool
     fields: Dict[str, FieldAnonymizer]
     _base_encryption_key = None
 
+    is_model: bool = True
```

Comment:
Is the type hinting needed here? this is not attribute declaration, but rather assignment. The attribute is declared in parent class.